package.json

```diff
@@ -19,7 +21,9 @@
   },
   "homepage": "https://github.com/tc39/proposal-record-tuple#readme",
   "dependencies": {
-    "ecmarkup": "^3.16.0",
+    "browser-sync": "^2.26.7",
+    "concurrently": "^5.2.0",
+    "ecmarkup": "^3.23.0",
```

Comment:
can we have those changes as a separate PR so I can use them in my work? Having the spec text reload in my browser seems like a cool thing

Reply:
@rricard sorry for the delay on this, I put these package.json changes on master, so if you rebase you should have them now.